packages/core/src/navigation/navigation.element.spec.ts

```diff
@@ -48,77 +48,17 @@ describe('cds-navigation', () => {
     expect(count).toBe(1);
   });
 
-  describe('should handle events', () => {
+  // TODO: figure out how to test the eventListeners added, dispatching events on the shadowDom elements behaves
```

Comment:
It is really hard to mimic events generated in the browser by user actions. Previously I was able to dispatch events on the component and they propagated down as expected. That doesn't happen and dispatching `tab` or `focus` events doesn't trigger the added eventListeners for the elements in the shadowDom. 

I spent 5-6 hours on this, again. (getting testing to work for the event listeners with the MVP release was tricky too). Coverage is still good with the changes but I've been unable to find a way to test the dual zone onKey changes for the root start and items container.

Reply:
To my understanding, events don't propagate down so much as bubble up. Did you try triggering the event on DOM children?

Reply:
I was using `element.dispatchEvent(...)` on the elements that we attach the event listeners on for the now dual zones. The event listeners used to be on the host and it was worked with `component.dispatchEvent(...)`